### PR TITLE
Update readme and pipfile

### PR DESCRIPTION
**Pull request by idanmiara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@
     - run: **python main.py all** to run both the generation and simulation.
     - run: **python main.py [simulate|generate] --help** for more help about each option.
     
+    
+## Installing and running under Linux (tested on Debian 10):
+- sudo apt install swig liblzma-dev libbz2-dev
+    - swig - building parasymoblic matrix needs swig
```
Comment (bentheiii) [inline README.md:45]: can't we pre-build the parasymbolic so that clients don't need swig to run it it?

File: README.md [hunk 2]
```diff
@@ -39,6 +39,15 @@
...
+    - liblzma-dev libbz2-dev - since we use Pandas, we need Python with support for bz2 and lzma support
+        - (https://stackoverflow.com/questions/22346269/bz2-is-module-not-available-when-installing-pandas-with-pip-in-python-virtual) 
+- Building Python 3.8 (currently not on the official repos...)
+    - https://linuxize.com/post/how-to-install-python-3-8-on-debian-10/
```
Comment (bentheiii) [inline README.md:49]: can't we just make a shell script that installs all this?

File: README.md (hunk 1 above)
Comment (idanmiara) [reply README.md:45]: Yes, I can push my build but I'm not sure it will work for every Linux...

File: README.md (hunk 2 above)
Comment (idanmiara) [reply README.md:49]: I could!

File: README.md (hunk 1 above)
Comment (bentheiii) [reply README.md:45]: you can push your CXX file and the clients won't need swig to build

**Comment by idanmiara** [comment]
back to the drawing board

**Pull request closed without merging**